modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/Activities/index.tsx

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import './index.scss';
+import dataActivities from './DataActivities';
+import TableList from './TableList';
+type TableHeaders = {
+	bold?: boolean;
+	key: string;
+	value: string;
+};
```

Comment:
You already created this TableHeaders type in this same commit, please change it to a general type. Try to export it from the Table component file.

Reply:
I removed the TableHeaders duplicated in this commit.